### PR TITLE
#986 The jars upload to worker should be moved to libs folder rather …

### PR DESCRIPTION
**Pull request by chia7712**
…than plugins folder

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #986)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/rest_interface.md [hunk 1]
```diff
@@ -1845,7 +1845,7 @@ The properties which can be set by user are shown below.
 1. clientPort (**int**) — worker client port.
 1. jmxPort (**int**) — worker jmx port.
 1. brokerClusterName (**string**) — broker cluster used to host topics for this worker cluster
-1. jars (**array(string)**) — the id of jars that will be loaded by worker cluster. You can require worker cluster to
+1. jarIds (**array(string)**) — the id of jars that will be loaded by worker cluster. You can require worker cluster to
```
Comment (saivirtue) [inline docs/rest_interface.md]: Good finding :)

**Comment by chia7712** [comment]
retry common

**Comment by chia7712** [comment]
retry configurator

**Comment by chia7712** [comment]
retry client

**Comment by chia7712** [comment]
retry common

**Comment by chia7712** [comment]
retry common

**Comment by chia7712** [comment]
retry agent

**Comment by chia7712** [comment]
retry client

**Comment by chia7712** [comment]
retry configurator

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
retry it

**Comment by chia7712** [comment]
The error is unrelated to this PR

**Pull request merged**